docs/tutorials/using-effective-hpa-to-scaling-with-effectiveness.md

```diff
@@ -184,3 +184,211 @@ status:
   expectReplicas: 0
 
 ```
+
+
+### Cron-bounded-driven autoscaling
+EffectiveHorizontalPodAutoscaler supports cron based autoscaling. Following is ehpa with cron spec.
+
+```yaml
+apiVersion: autoscaling.crane.io/v1alpha1
+kind: EffectiveHorizontalPodAutoscaler
+metadata:
+  name: php-apache-local
+spec:
+  # ScaleTargetRef is the reference to the workload that should be scaled.
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: php-apache
+  minReplicas: 1        # MinReplicas is the lower limit replicas to the scale target which the autoscaler can scale down to.
+  maxReplicas: 10       # MaxReplicas is the upper limit replicas to the scale target which the autoscaler can scale up to.
+  scaleStrategy: Auto   # ScaleStrategy indicate the strategy to scaling target, value can be "Auto" and "Manual".
+  # Better to setting cron to fill the one complete time period such as one day, one week
+  # Below is one day cron scheduling, it
+  #100                  --------     ---------        ----------
+  #                    |       |    |        |       |         |
+  #1        ------------       -----         --------          ----------
+  #(time)   0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23
+  # Local timezone means you use the server's(or maybe is a container's) timezone which the craned running in. for example, if your craned started as utc timezone, then it is utc. if it started as Asia/Shanghai, then it is Asia/Shanghai.
+  crons:
+    - name: "cron1"
+      timezone: "Local"
+      description: "keep normal"
+      start: "0 0 ? * *"
+      end: "0 6 ? * *"
+      targetReplicas: 1
+    - name: "cron2"
+      timezone: "Local"
+      description: "scale up"
+      start: "0 6 ? * *"
+      end: "0 9 ? * *"
+      targetReplicas: 100
```

Comment:
maxReplicas is 10, if it will limit the target replicas? like even user specify target replicas as 100, the actual number will be 10 which is limited by maxReplicas?